dev/com.ibm.websphere.appserver.features/visibility/public/jaxb-2.3/com.ibm.websphere.appserver.jaxb-2.3.feature

```diff
@@ -4,24 +4,13 @@ visibility=public
 singleton=true
 IBM-App-ForceRestart: uninstall, \
  install
-IBM-API-Package: \
```

Comment:
If we don't export the packages for the application to use then how are they made available to the customer application?

Reply:
Undid this in my last commit